tests/roles/development_environment/tasks/main.yaml

```diff
@@ -1,10 +1,13 @@
 - name: pre-launch test VM instance
   no_log: "{{ use_no_log }}"
   when: prelaunch_test_instance|bool
+  vars:
+    cinder_backend_configured: "{{  cinder_volume_backend == 'ceph' and cinder_backup_backend == 'ceph' }}"
```

Comment:
Wondering if it would be better to use `cinder_volume_backend == '' or cinder_backup_backend == ''`? If we add testing for another backend like NFS, this wouldn't have to change then. Similarly with the `cinder_verify.yaml`, perhaps we should be checking `cinder_volume_backend != ''` rather than `cinder_volume_backend == 'ceph'` ?

Reply:
is the cinder_backup_backend validation needed? If we don't have cinder_volume configured then we can't create volumes and eventually can't create backups as well.
Also agree that when adding jobs for other backends like NFS or iSCSI this might need change so we should not stick to ceph being the only valid backend available for cinder.

Reply:
I found a better solution, which is adding a var that we can use differently upstream and downstream